template/.phrase.yml

```diff
@@ -47,3 +37,14 @@ phrase:
           tags: error,rating
```

Comment:
We should take out the "rating" tag for not including common rating messages in the template translations